libs/gi/dm-localization/assets/locales/chs/artifact_FinaleOfTheDeepGalleries_gen.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
I know we won't be getting non-english translations from hakushin, but can we opt to not generate files in these languages?

Reply:
true, i didn't think the fallback worked liek that